internal/consensus/state_test.go

```diff
@@ -1857,20 +2389,13 @@ func TestStateSlashingPrevotes(t *testing.T) {
 	// add one for a different block should cause us to go into prevote wait
 	hash := rs.ProposalBlock.Hash()
 	hash[0] = byte(hash[0]+1) % 255
-	signAddVotes(cs1, tmproto.PrevoteType, hash, rs.ProposalBlockParts.Header(), vs2)
-
-	<-timeoutWaitCh
-
-	// NOTE: we have to send the vote for different block first so we don't just go into precommit round right
-	// away and ignore more prevotes (and thus fail to slash!)
-
-	// add the conflicting vote
-	signAddVotes(cs1, tmproto.PrevoteType, rs.ProposalBlock.Hash(), rs.ProposalBlockParts.Header(), vs2)
+	signAddVotes(tmproto.PrevoteType, hash, config.ChainID(), blockID,
+		rs.ProposalBlock.Hash(), config.ChainID(), rs.ProposalBlockParts.Header(), vs2)
 
 	// XXX: Check for existence of Dupeout info
 }
 
-func TestStateSlashingPrecommits(t *testing.T) {
+func TestStateSlashing_Precommits(t *testing.T) {
```

Comment:
Does it make sense to just delete these commented out tests? I can't imagine that any part of them still works?

Reply:
Yes.